src/components/LandingExplain.coffee

```diff
@@ -163,6 +161,7 @@ module.exports = React.createClass
         div null, "PR & Marketing"
       div className: "teamMember",
         div className: "circle blue"
+          img className: "teamImage", src: require "../img/team/placeholder_4.png"
```

Comment:
Siehe meine Email von heute morgen...

LG 

Thao 

> On 11 Jul 2016, at 01:47, Markus Kohlhase <notifications@github.com> wrote:
> 
> In src/components/LandingExplain.coffee:
> 
> > @@ -163,6 +161,7 @@ module.exports = React.createClass
> >          div null, "PR & Marketing"
> >        div className: "teamMember",
> >          div className: "circle blue"
> > +          img className: "teamImage", src: require "../img/team/placeholder_4.png"
> There is no placeholder_4.png file. Would you mind to add it? Thx.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>